how-to/add_k8s_cloud.rst

```diff
@@ -0,0 +1,81 @@
+JAAS: Add a K8S cloud to JIMM
+=============================
+
+Introduction
+------------
+In this how-to, we will demonstrate adding a K8S cloud to JIMM
+and deploying a model.
+
+Prerequisites
+-------------
+
+For this how-to, you will require the following:
+
+- Basic knowledge of Juju
+- Have JIMM deployed with admin access
+- Have a machine based controller added to JIMM
+- Basic knowledge of K8S
+- A controller running in the region next to your K8S deployment, whether that be
```

Comment:
This is confusing, `in the region next to your k8s deployment`. What does "next to" mean?

Reply:
Ah ok I meant same subnet or a subnet with specific routing rules to another subnet, I'll update to be clearer

Reply:
updated